components/support/nimbus-fml/src/backends/swift/gen_structs/mod.rs

```diff
@@ -83,6 +83,11 @@ impl<'a> FeatureManifestDeclaration<'a> {
         let oracle = &self.oracle;
         // Filter out our own module.
         let my_module = &self.fm.about.nimbus_module_name();
+        // Get the app-services module from an environment variable.
+        // If it doesn't exist, then we don't add it.
+        let as_module = std::env::var("MOZ_APPSERVICES_MODULE")
```

Comment:
Should this variable be documented somewhere? (I'm guessing documented somewhere in the iOS app)